public/checkDb.js

```diff
@@ -0,0 +1,50 @@
+import data from './db.json'  assert { type: 'json' };
```

Comment:
1. there is an error without this `assert` command.
`TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "file:///.../aia-catalog/public/db.json" needs an import attribute of type "json"`

2. Otherwise, there is the ESLint error
`ESLint: Parsing error: Unexpected token assert`